bazel/external/wee8.BUILD

```diff
@@ -0,0 +1,24 @@
+licenses(["notice"])  # Apache 2
```

Comment:
I thought you're working on bazel native build, where is it? :)

Reply:
I am, but it's not my top priority at this moment.